src/Sets/Interval.jl

```diff
@@ -586,3 +586,21 @@ The interval obtained by applying the numerical scale to the given interval.
 function scale(α::N, x::Interval{N}) where {N<:Real}
     return Interval(α * x.dat)
 end
+
+"""
+    center(H::Interval{N}, i::Int) where {N<:Real}
+
+Return the center along a given dimension of a interval.
+
+### Input
+
+- `x` -- interval
+- `i` -- dimension of interest
+
+### Output
+
+The center along a given dimension of the interval.
+"""
+@inline function center(x::Interval{N}, i::Int) where {N<:Real}
+    return center(x)[i]
```

Comment:
This unnecessarily creates a temporary array, i would suggest to implement it like so

```julia
@inline function center(x::Interval{N}, i::Int) where {N<:Real}
     @boundscheck i == 1
     return IntervalArithmetic.mid(x.dat)
end
```